server/src/entities/exif.entity.ts

```diff
@@ -25,8 +26,8 @@ export class ExifEntity {
   @Column({ type: 'bigint', nullable: true })
   fileSizeInByte!: number | null;
 
-  @Column({ type: 'varchar', nullable: true })
-  orientation!: string | null;
+  @Column({ type: 'enum', enum: Orientation, nullable: true })
```

Comment:
For this case, the keys are basically already enum values. It'd be nice to make this a `"char"` (not `char`) and use a check constraint that it's between 1-8. It's 1 byte instead of 4 and doesn't have the string vs number ambiguity that an enum would have.